src/fluree/server/consensus/watcher.clj

```diff
@@ -0,0 +1,61 @@
+(ns fluree.server.consensus.watcher
+  "When mutation requests happen over http (e.g. creating a new ledger), we want
+  to wait for the operation to complete through consensus before providing the
+  response back to the API requester. These operations get delivered via
+  consensus, where they end up being picked up by a responsible server from a
+  queue, processed, and then the results broadcast back out."
```

Comment:
I like ns docstrings and I think we should use them more often, but this one kind of feels like it's missing the part about what this ns does. I think it's good to keep what's here as an example use case, but would love to have it start with a short explanation of what this ns is for in the abstract.

Reply:
That makes sense. This was a raw comment at the top of the ns and I just copied it verbatim into the ns docstring instead. I'll try to add a description of what this ns does as well.